packages/docs/integrate/custom-integration.md

```diff
@@ -0,0 +1,334 @@
+# Creating a simple custom integration
+
+::: warning Want to build an integration?
+If you want to integrate with Vue Storefront, don't hesitate to get in touch with the core team on our [Discord](https://discord.vuestorefront.io/) server. We are eager to help you to ensure its high quality and maybe even officially recommend it 😉
+:::
+
+Vue Storefront provides out-of-the-box integrations for various third-party solutions (you can find them listed [here](https://docs.vuestorefront.io/v2/integrations/)). However, sometimes your project might need to connect to a service that Vue Storefront does not integrate with by default.
+
+For this case, you can leverage our default pattern to create your own integration easily and keep your project coherent!
```

Comment:
Avoid using the words "easy" or "simple" (document title). What seems easy to you might not be for others.

https://www.knowledgeowl.com/home/dont-say-simply-jim-fisher